addon/components/frost-action-bar.js

```diff
@@ -26,7 +26,10 @@ export default Component.extend({
     selectedItems: PropTypes.arrayOf(PropTypes.oneOfType([
       PropTypes.EmberObject,
       PropTypes.object
-    ])).isRequired
+    ])).isRequired,
+
+    // callbacks
+    getSelectedItemsLabel: PropTypes.func
```

Comment:
What do you think of creating a special prefix for anything intended strictly for i18n?  E.g.

```
i18n: PropTypes.shape({
  selectedItems: PropTypes.func
})
```

Then when I'm looking at the interface I know exactly what it's intended for and the "translation" portion of the interface is quite clear